src/sentry/workflow_engine/endpoints/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import re_path
+
+from .project_detector_index import ProjectDetectorIndexEndpoint
+
+urlpatterns = [
```

Comment:
:clap: We should do more URL files, the current big list is a pain to work with.